gulpfile.js

```diff
@@ -1,51 +1,68 @@
-const gulp = require('gulp');
-const sass = require('gulp-sass');
-const connect = require('gulp-connect');
-var rename = require('gulp-rename');
+const gulp = require("gulp");
+const sass = require("gulp-sass");
+const connect = require("gulp-connect");
+const imagemin = require("gulp-imagemin");
+var rename = require("gulp-rename");
 
-gulp.task('sass', function() {
+gulp.task("imagemin", function() {
+  gulp.src(["./src/imgs/*.jpg"]).pipe(gulp.dest("./srv/imgs/"));
```

Comment:
You're not actually using `imagemin`. What you're doing atm is just piping your images from one folder to the other. 
```suggestion
  gulp.src(["./src/imgs/*.jpg"]).pipe(imagemin()).pipe(gulp.dest("./srv/imgs/"));
```